# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../netsniff-ng/0004-fix-include-poll.h.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

The standard defines <poll.h> and not <sys/poll.h>

http://pubs.opengroup.org/onlinepubs/009696799/basedefs/poll.h.html

--- a/ring.h	2014-04-07 22:03:43.038037257 +0200
+++ b/ring.h	2014-04-07 22:04:06.298592881 +0200
@@ -20,7 +20,6 @@
 #include <sys/ioctl.h>
 #include <string.h>
 #include <poll.h>
-#include <sys/poll.h>
 
 #include "built_in.h"
 #include "die.h"
--- a/curvetun_client.c	2014-04-07 22:05:50.841029712 +0200
+++ b/curvetun_client.c	2014-04-07 22:06:21.713719262 +0200
@@ -20,7 +20,7 @@
 #include <sys/socket.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <netinet/tcp.h>
 #include <netinet/udp.h>
 #include <linux/if_tun.h>
--- a/curvetun_server.c	2014-04-07 22:05:56.921159753 +0200
+++ b/curvetun_server.c	2014-04-07 22:06:34.045997332 +0200
@@ -20,7 +20,7 @@
 #include <netinet/in.h>
 #include <netinet/tcp.h>
 #include <netinet/udp.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/wait.h>
